keybuffer.h

```diff
@@ -32,3 +33,4 @@ void                    st_key_buffer_reset(st_key_buffer_t *buf);
 void                    st_key_buffer_push(st_key_buffer_t *buf, uint16_t keycode);
 void                    st_key_buffer_pop(st_key_buffer_t *buf, uint8_t num);
 void                    st_key_buffer_print(st_key_buffer_t *buf);
+void                    st_key_buffer_get_context_string(st_key_buffer_t *buf, char* output_string, uint8_t context_len);
```

Comment:
Should be: `st_key_buffer_get_context_string(const st_key_buffer_t *buf, char *output_string, uint8_t context_len);`